Modules/_asynciomodule.c

```diff
@@ -3944,6 +3946,7 @@ static int
 module_exec(PyObject *mod)
 {
     asyncio_state *state = get_asyncio_state(mod);
+    Py_SET_REFCNT(&state->asyncio_tasks.tail, _Py_IMMORTAL_REFCNT);
```

Comment:
Use `_Py_SetImmortal` to mark objects as immortal.

Reply:
Done, actually I tried to find the function but I forgot the name so did it manually